shell/ci/testing/setup-devenv.sh

```diff
@@ -29,9 +29,24 @@ if [[ -n $CI ]]; then
 
   if ! command -v kubectl >/dev/null; then
     info "Installing kubectl"
-    # Taken from: https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/#install-using-native-package-management
-    sudo curl -fsSLo /etc/apt/keyrings/kubernetes-archive-keyring.gpg https://packages.cloud.google.com/apt/doc/apt-key.gpg
-    echo "deb [signed-by=/etc/apt/keyrings/kubernetes-archive-keyring.gpg] https://apt.kubernetes.io/ kubernetes-xenial main" | sudo tee /etc/apt/sources.list.d/kubernetes.list
+    tmpFile=$(mktemp)
+    keyringLocation=/etc/apt/keyrings/kubernetes-archive-keyring.gpg
+
+    curl -fsSLo "$tmpFile" https://packages.cloud.google.com/apt/doc/apt-key.gpg
+
+    # 2023-05-23: GCP changed their key to be armored and their location. Currently
```

Comment:
This is weird wording. Do you mean this?
```suggestion
    # 2023-05-23: GCP changed their key to be armored and changed the key location. Currently
```